net/browserless_agent/tests/graphData.ts

```diff
@@ -0,0 +1,90 @@
+import { GraphData } from "graphai";
+
+// Graph for testing content retrieval
+export const graphDataContent: GraphData = {
+  version: 1,
```

Comment:
This version refers to the graph data version and is different from the GraphAI version. Currently, it is 0.5, but we usually use `graphDataLatestVersion`.